src/dune_rules/main.ml

```diff
@@ -75,6 +75,32 @@ let init
       let download ~rule_digest:_ = Fiber.return ()
     end)
   in
+  let cache_config =
```

Comment:
It seems odd for this code to live in `src/dune_rules`, since these aren't rules. Maybe it should be in the command line somewhere.

Reply:
(Not a blocker for this PR, just an observation.)